src/components/RocketsDependencies/Rocket.js

```diff
@@ -0,0 +1,60 @@
+
+import React from "react";
+import PropTypes from 'prop-types';
+import { ListGroup, Badge, Image, Col, Row }  from "react-bootstrap";
+import { useDispatch , useSelector } from "react-redux";
+import ReserveButton from "./ReserveButtonComponent";
+import { toggleReservationAction } from "../../redux/rockets/rockets";
+
+const visibleBadge = {
+  display: "inline-block",
+}
+
+const invisibleBadge = {
+  display: "none",
+}
+
+const Rocket = ({ rocketData }) => {
+  const reservations = useSelector(state => state.rocketsReducer.rockets);
+  const thisReservation = reservations.find(reservation => reservation.id === rocketData.id);
```

Comment:
- [ ] [OPTIONAL] would recommend removing these 2 lines since `thisReservation` is the same object as `rocketData`.